response_operations_ui/common/mappers.py

```diff
@@ -22,7 +23,9 @@ def convert_events_to_new_format(events):
         formatted_events[event['tag']] = {
             "day": day,
             "date": date,
-            "time": time
+            "month": str(date_time.month) if len(str(date_time.month)) == 2 else f"0{date_time.month}",
```

Comment:
I feel like the way datetime is being formatted in this function is slightly weird.

calendar.month_name[date_time.month][:3] ---> date_time.strftime('%b')
f"{date_time.strftime('%d')} {month} {date_time.strftime('%Y')}" ---> date_time.strftime('%d %b %Y')
f"{date_time.strftime('%H:%M')} GMT" ---> date_time.strftime('%H:%M GMT')
str(date_time.month) if len(str(date_time.month)) == 2 else f"0{date_time.month}" ---> date_time.strftime('%m')?